Fakeドライバー+インメモリストアでのレースコンディション

CI実行時にエラーとなる。

エラー内容: 

```
=== CONT  TestBuilder_Build_BlackBox
    crud.go:250: Update is failed:  Error in response: &sacloud.APIErrorResponse{IsFatal:true, Serial:"", Status:"409 Conflict", ErrorCode:"409", ErrorMessage:"request to Server[ID:100000000009] is conflicted: Shutdown is failed"}
--- FAIL: TestBuilder_Build_BlackBox (0.27s)
FAIL
FAIL	github.com/sacloud/libsacloud/v2/helper/builder/server	0.513s
```

おそらくFakeドライバーの並列実行時の問題だと思われるが、影響は小さいため一旦無視する。

--- a/v2/helper/api/caller.go
+++ b/v2/helper/api/caller.go
@@ -166,4 +166,8 @@ func SetupFakeDefaults() {
 	defaults.DefaultPowerHelperShutdownRetrySpan = defaultInterval
 	defaults.DefaultPowerHelperInitialRequestRetrySpan = defaultInterval
 	defaults.DefaultPowerHelperInitialRequestTimeout = defaultInterval * 100
+
+	fake.PowerOnDuration = time.Millisecond
+	fake.PowerOffDuration = time.Millisecond
+	fake.DiskCopyDuration = time.Millisecond
 }

--- a/v2/sacloud/fake/timer.go
+++ b/v2/sacloud/fake/timer.go
@@ -137,7 +137,7 @@ func startPowerOn(resourceKey, zone string, readFunc func() (interface{}, error)
 
 func startPowerOff(resourceKey, zone string, readFunc func() (interface{}, error)) {
 	counter := 0
-	ticker := time.NewTicker(PowerOnDuration)
+	ticker := time.NewTicker(PowerOffDuration)
 	go func() {
 		defer ticker.Stop()
 		for {

